Library/Formula/domoticz.rb

```diff
@@ -0,0 +1,66 @@
+class DomoticzDownloadStrategy < SubversionDownloadStrategy
+  def stage
+    # Domoticz' build uses the svn revision for its version.
+    # Thus rather than the default behavior of exporting the working copy
+    # we must copy it including .svn folders.
+    cp_r "#{cached_location}/.", Dir.pwd
+  end
+end
+
+class Domoticz < Formula
+  homepage "http://www.domoticz.com/"
+  head "http://svn.code.sf.net/p/domoticz/code/trunk/", :using => DomoticzDownloadStrategy
```

Comment:
Created a PR on Homebrew/homebrew-head-only for this formula.

https://github.com/Homebrew/homebrew-head-only/pull/92